rdkit/Chem/Draw/qtCanvas.py

```diff
@@ -10,7 +9,10 @@
 #
 
 from rdkit.Chem.Draw.canvasbase import CanvasBase
-from PySide import QtGui, QtCore
+try:
+  from PySide import QtGui, QtCore
+except ImportError:
+  from PyQt5 import QtGui, QtCore
```

Comment:
Have you tested with PyQt5? It's great if this works, but I'd want to be sure someone had actually tested it.